examples/Boost-python-numpy/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2013, Ruslan Baratov
+# Modified work: Copyright (c) 2018, Gregory Kramida
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+option(HUNTER_BUILD_SHARED_LIBS "..." ON)
+
+# Configure:
+set(PYTHON_VERSION 3.5)
+set(TESTING_CONFIG_OPT FILEPATH "${CMAKE_CURRENT_LIST_DIR}/config.cmake")
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-boost)
+
+# Requires python version 3.5. Change this in config.cmake if necessary.
```

Comment:
So the version is optional but defaults to 3.5?

Reply:
*shrug* no idea, I copied the existing Boost-python test. To run it locally I have to use 37 (as I have python 3.7 installed)

Reply:
3.5 is what is available on Windows Appveyor and Travis Linux, which is why it is the default here and in Boost-Python.